views/items/index.pug

```diff
@@ -20,12 +20,15 @@ block content
   div.main-page
     .work-page
       .work-page__main
-        .icon.icon__star
+        .icon.icon__star(data-id=root.id, data-starred=`${root.starred}`, class={starred: root.starred})
```

Comment:
@jrob8577 We're currently stumped on how to conditionally assign the class "star" OR "star--starred" using logic similar to the above. We weren't able to track down the type of syntax being used here: `class={starred: root.starred}`  Any tips? Thanks!